CSS/quiz.css

```diff
@@ -0,0 +1,28 @@
+@import url(./rule.css);
+.question-title{
+    font-size: 1.6rem;
+}
+.q-list-wrapper{
```

Comment:
maybe you may give a bit more descriptive name for better understanding like `ques-list-wrapper`

Reply:
ok